lib/tasks/sys_admin_tasks.rake

```diff
@@ -0,0 +1,7 @@
+desc "Tasks triggered by sys admins"
+namespace :sys_admin_tasks do
+  desc "destroy user"
+  task :destroy_user, [:email] => :environment do |_, args|
+    User.find_by(email: args.email).destroy!
+  end
```

Comment:
If they have the wrong email, it will show then an exception.  If this was going to be used by a non ruby programmer, it might be nicer to catch the exception and say "Email not found" on the console

A nice to have